libs/ui/src/system_call_api.tsx

```diff
@@ -45,6 +47,17 @@ function createReactQueryApi(getApiClient: () => SystemCallApiClient) {
         return useMutation(apiClient.exportLogsToUsb);
       },
     },
+    getBatteryInfo: {
+      queryKey(): QueryKey {
+        return ['getBatteryInfo'];
+      },
+      useQuery() {
+        const apiClient = getApiClient();
+        return useQuery(this.queryKey(), () => apiClient.getBatteryInfo(), {
+          refetchInterval: BATTERY_POLLING_INTERVAL_GROUT,
+        });
+      },
+    },
```

Comment:
I'm adding this here so it's accessible in all the apps, but will only be active if the hook is mounted. So will use in VxScan but will not affect the other apps.